### ISSUE TITLE
Handler not found: POST /VolumeDriver.Capabilities

### ISSUE THREAD

**Issue by gotliuyan** [rancher/convoy#224]
``` **ERRO[0066] Handler not found: POST /VolumeDriver.Capabilities**  pkg=daemon
DEBU[0078] Handle plugin get volume: POST /VolumeDriver.Get  pkg=daemon
DEBU[0078] Request from docker: &{huihigh map[]}         pkg=daemon
DEBU[0078]                                               event=mountpoint object=volume pkg=daemon reason=prepare volume=huihigh
DEBU[0078]                                               event=mountpoint mountpoint= object=volume pkg=daemon reason=complete volume=huihigh
DEBU[0078] Found volume huihigh for docker               pkg=daemon
DEBU[0078] Response:  {
	"Volume": {
		"Name": "huihigh"
	}
}  pkg=daemon```

**Comment by unglaublicherdude** [comment]
I have the Same problem. This log event is bloating our syslogs.

```
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.537049359+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.540329313+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.540612702+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.542310363+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.550172235+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.556334853+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.571038513+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.598494329+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
Aug 24 09:50:13 queue-001 dockerd[1073]: time="2018-08-24T09:50:13.613025943+02:00" level=warning msg="Volume driver convoy returned an error while trying to query its capabilities, using default capabilities: VolumeDriver.Capabilities: Handler not found: POST /VolumeDriver.Capabilities"
```

**Comment by pwFoo** [comment]
Hi @yasker would be nice to get a fix for that issue. 
Set up convoy and it seems to work fine with first tests.

**Comment by yasker** [comment]
@pwFoo Fixed in v0.5.1 release.
